app/components/questions/RentBurdenRates.js

```diff
@@ -0,0 +1,216 @@
+            textAnchor={x > cx ? "start" : "end"}
+            dominantBaseline="central">
+        {`${valueFormatted}`}
+      </text>
+    );
+  };
+  
+  
+  return (
+    <ResponsiveContainer width="100%" height={400}>
+      <PieChart
+        width={400}
+        height={400}
+        margin={{ bottom: 30 }}
+      >
+        <Pie
+          data={data}
+          nameKey="label"
+          dataKey="value"
+          label={renderCustomizedLabel}
+          cx="50%"
+          cy="50%"
+          fill=""
+          outerRadius={150}
+        >
+          {data.map((entry, i) => (
+            <Cell
+              key={`cell-${i}`}
+              fill={entry.color}
+            />
+          ))}
+        </Pie>
+        <Tooltip content={<CustomTooltip />} />
+        <Legend
+          layout="horizontal"
+          verticalAlign="top"
+          align="center"
+          wrapperStyle={{ bottom: -10 }}
+        />
+      </PieChart>
+    </ResponsiveContainer>
+  );
+}
+
+function AreaSelector(props) {
+  if(!props.data){
+    return (
+      <p>Waiting for data...</p>
+    )
+  }
+  
+  const handleChange = (e) => props.handleChange(parseInt(e.target.value))
+  
+  return (
+    <div>
+      <p>Select an Area of Interest</p>
+      <select onChange={handleChange}>
+        {props.data.metrics.map((d, i) => (
+          <option key={i} value={d.area_number}>{d.name}</option>
+        ))}
+      </select>
+    </div>
+  );
+}
+
+function AreaRentBurdenedMessage(props) {
+  const { areaData } = props; 
+  
+  if(!areaData){
+    return (
+      <p>Waiting for data...</p>  
+    )
+  }
+  
+  if(!areaData.rent_burdened_2019){
+    return (
+      <span>There is no data for {areaData.name}</span>
+    )
+  } else {
+    return (
+      <span>Based on the data from 2019, {areaData.rentBurdenRateFormatted} of households in {areaData.name} are rent burdened</span>
+    )
+  }
+}
+
+export default function RentBurdenedRates(props) {
+  const { loading, error, data } = useFetch(ENDPOINT, {
+    method: "POST",
+    body: JSON.stringify({metrics: ["rent_burdened_2019"]})
+  }, []);
+  const [selectedArea, setSelectedArea] = useState(1)
+  const { chartData, areaData } = transformData(data, error, selectedArea);
+  
+  useEffect(() => {
+    props.setContentIsLoading(loading);
+  }, [loading]);
+  
+  return (
+    <div>
+      <div className="center medium-width">
+        <h2>Rent Burdened Households Rate by Area</h2>
+        <AreaSelector data={data} handleChange={setSelectedArea} />
+        <p>What percentage of households in {areaData.name} are rent burdened?</p>
+        <FailureNotification error={error} data={data} />
+      </div>
+      <div className="center">
+        <ExamplePieChart data={chartData} />
+        <br />
+      </div>
+      <div className="center medium-width">
+        <p>
+        <span>Based on data from 2019, {areaData.rentBurdenRateFormatted} of households in {areaData.name} are rent burdened.</span>
```

Comment:
Oh I realized just now that you're not even using the AreaRentBurdenedMessage element you have defined above. In a later PR you can replace this HTML with that component.